examples/py/build-ohlcv-bars.py

```diff
@@ -11,7 +11,7 @@
 
 # ----------------------------------------------------------------------------
 import asyncio
-import ccxt.async_support as ccxt  # noqa: E402
+import ccxt.pro as ccxt  # noqa: E402
```

Comment:
@pcriadoperez I'm not sure that we should default to the `pro` module, what if users want to use an exchange that does not have a WS implementation?

Reply:
Good Point! I put a check to see if ccxt.pro is used to choose one or the other